client/app/components/dialog-editor-modal-field-template/dynamic.html

```diff
@@ -0,0 +1,35 @@
+<!-- any dynamic element -->
+<div pf-form-group pf-label="{{'Entry Point'|translate}}">
+  <!-- TODO: until treepicker and API for it are in the SSUI -->
```

Comment:
Aah, thanks Alberto, seems that `http://localhost:3000/api/automate?depth=-1&attributes=name,klass` does what we need :)